src/devices/Hmc5883l/Hmc5883l.cs

```diff
@@ -120,5 +138,33 @@ public void Dispose()
                 _sensor = null;
             }
         }
+
+        /// <summary>
+        /// Reads device statuses .
+        /// </summary>
+        /// <returns>IList<Status></returns>
```

Comment:
the return comment should be more specific mentioning what is it returning and perhaps even possible values and their meaning

Reply:
Fixed